plugins/vhd.plugin

```diff
@@ -0,0 +1,88 @@
+# molior-deploy plugin
+# vim: syntax=sh
+
+preinit_deployment_vhd()
+{
+  REQUIRED_HOST_COMMANDS="kpartx qemu-img qemu-nbd"
+  INSTALL_GRUB="yes"
+  PARTSEP="p"
+}
+
+init_deployment_vhd()
+{
+  vmname=$(get_deployment_filename "vhd")
+  deployment=$vmname
+
+  # FIXME check VMSIZE
+  echo " * creating VHD disk of ${VMSIZE}M"
+  VMDISK=$WORK_DIR/$deployment
+  qemu-img create -f vpc -o subformat=fixed,force_size $VMDISK ${VMSIZE}M >&2
+  if [ $? -ne 0 ]; then
+    log_error "Error creating $VMDISK"
+  fi
```

Comment:
Should we use `exit_on_error` instead?

Reply:
yes, but will make a separate PR, for changing everything :)

Reply:
Okay, thanks.